oap-server/server-starter/src/main/resources/otel-rules/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -0,0 +1,71 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This will parse a textual representation of a duration. The formats
+# accepted are based on the ISO-8601 duration format {@code PnDTnHnMn.nS}
+# with days considered to be exactly 24 hours.
+# <p>
+# Examples:
+# <pre>
+#    "PT20.345S" -- parses as "20.345 seconds"
+#    "PT15M"     -- parses as "15 minutes" (where a minute is 60 seconds)
+#    "PT10H"     -- parses as "10 hours" (where an hour is 3600 seconds)
+#    "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+#    "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+#    "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+#    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+#    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+# </pre>
+
+filter: "{ tags -> tags.job_name == 'bookkeeper-monitoring' }" # The OpenTelemetry job name
+expSuffix: tag({tags -> tags.cluster = 'bookkeeper::' + tags.cluster}).service(['cluster'], Layer.BOOKKEEPER)
+metricPrefix: meter_bookkeeper
+
+# Metrics Rules
+metricsRules:
+
+  # storage metrics
+  - name: bookie_ledgers_count
+    exp: bookie_ledgers_count.sum(['cluster', 'node'])
+  - name: bookie_ledger_writable_dirs
+    exp: bookie_ledger_writable_dirs.sum(['cluster', 'node'])
+  - name: bookie_ledger_dir_data_bookkeeper_ledgers_usage
+    exp: bookie_ledger_dir_data_bookkeeper_ledgers_usage.sum(['cluster', 'node'])
+  - name: bookie_entries_count
+    exp: bookie_entries_count.sum(['cluster', 'node'])
+
+  - name: bookie_write_cache_size
+    exp: bookie_write_cache_size.sum(['cluster', 'node'])
+  - name: bookie_write_cache_count
+    exp: bookie_write_cache_count.sum(['cluster', 'node'])
+
+  - name: bookie_read_cache_size
+    exp: bookie_read_cache_size.sum(['cluster', 'node'])
+  - name: bookie_read_cache_count
+    exp: bookie_read_cache_count.sum(['cluster', 'node'])
+
+  - name: bookie_write_cache_hits
+    exp: bookie_write_cache_hits.sum(['cluster', 'node']).rate('PT1M')
+  - name: bookie_read_cache_hits
+    exp: bookie_read_cache_hits.sum(['cluster', 'node']).rate('PT1M')
+  - name: bookie_write_cache_misses
+    exp: bookie_write_cache_hits.sum(['cluster', 'node']).rate('PT1M')
```

Comment:
should be `bookie_write_cache_misses.sum(['cluster', 'node']).rate('PT1M')`

Reply:
This should be deleted.